src/test/java/service/impl/DataConverterImplTest.java

```diff
@@ -0,0 +1,110 @@
+package service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import model.FruitTransaction;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import service.DataConverter;
+
+class DataConverterImplTest {
+    private static final String EMPTY_LINES_FILE = "src/main/java/resources/fileWithEmptyLines.csv";
```

Comment:
You should have a separate resources directory in the test package and use separate files for testing.